cmd/ocm/create/user/cmd.go

```diff
@@ -121,10 +123,15 @@ func run(cmd *cobra.Command, argv []string) error {
 			Send()
 		if err != nil {
 			fmt.Printf("Failed to add '%s' user '%s' to cluster '%s': %v\n", args.group, username, clusterKey, err)
+			failedToAddUser, addUserErr = true, err
```

Comment:
If there will be multiple failures the last one will override the ones before.
Try to return a list of the users that failed:

```
return fmt.Errorf("Failed to create the following users in group '%s': user1, user4", args.group"
```